ichnaea/data/public.py

```diff
@@ -76,9 +78,7 @@ def write_stations_to_csv(session, path, today,
                             cellid=min_cellid
                         )).fetchall()
                     if rows:
-                        buf = '\r\n'.join([row.cell_value for row in rows])
-                        if buf:
-                            buf += '\r\n'
+                        buf = ''.join(row.cell_value + linesep for row in rows)
```

Comment:
It took me a minute to figure this out (I had to go back and understand ``row.cell_value``), but it looks like it will do the same thing as the previous code.